### ISSUE TITLE
Create a trailer for SMA

### ISSUE THREAD

**Issue by alexis-** [supermemo-wiki/Issues#4]
Short clips of each of these functions are needed:

Theme: Carbon(?)

- [ ] PDF
  - [ ] Text extract (simply extract piece of text)
  - [ ] Image extract
  - [ ] Snapshot area extract
  - [ ] Multiple selection
  - [ ] OCR
  - [ ] Dictionary
- [ ] LaTeX
  - [ ] LaTeX -> Image
  - [ ] Image -> LaTeX
- [ ] Import
  - [ ] Browser
  - [ ] Feeds
  - [ ] Website configs (filters)
- [ ] Dictionary

**Comment by dumbyme** [comment]
Do you mean a 'Demo' of these functions?

**Comment by alexis-** [comment]
Yes. A glimpse of each feature, without voice that people can watch in a jiff. @nikedinikedi is already on it
